verl/single_controller/base/worker.py

```diff
@@ -130,6 +130,35 @@ def _query_collect_info(self, mesh_name: str):
         assert mesh_name in self.__collect_dp_rank, f"{mesh_name} is not registered in {self.__class__.__name__}"
         return self.__collect_dp_rank[mesh_name]
 
+    def get_dispatch_collect(self):
+        """Get all registered dispatch and collect dp_ranks.
+
+        Returns:
+            dict[str, int]:
+                A dictionary mapping mesh names to their dispatch dp_ranks.
+            dict[str, bool]:
+                A dictionary mapping mesh names to whether they are used for collect.
+        """
+        return self.__dispatch_dp_rank, self.__collect_dp_rank
+
+    def set_dispatch_collect(self, dispatch_dp_rank: dict[str, int], collect_dp_rank: dict[str, bool]):
```

Comment:
Give the new dispatch collect a new name is more general

Reply:
`get_dispatch_collect` and `set_dispatch_collect` are pair function to mirror inner worker's dispatch info to outer worker. For example, `ActorRolloutRefWorker` is composed of two `ActorWorker`(actor, ref).

Reply:
Add `mesh_name` to override origin mesh_name when `set_dispatch_collect`.